x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/scalar/spatial/AnyCombiner.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.esql.expression.function.scalar.spatial;
+
+import org.elasticsearch.compute.ann.MvCombiner;
+
+/**
+ * This multivalue combiner supports combining boolean types, where any true value will result in a true result.
+ */
+public class AnyCombiner implements MvCombiner<Boolean> {
```

Comment:
I'd stick these next to `MvCombiner` over in the `compute` engine.

Reply:
Each attempt caused dependency issues, so I've left it where it is for now.